masschallenge/django-accelerator
# [AC-9679]: Create migration to add Communities and link Program family to Communities

[AC-9679](https://masschallenge.atlassian.net/browse/AC-9679)

**How to test:**
Ensure the definition of done on [AC-9679](https://masschallenge.atlassian.net/browse/AC-9679)
 are met
